src/interiorpointsolver.jl

```diff
@@ -1224,179 +1139,179 @@ is_valid(args...) = all(is_valid(arg) for arg in args)
 
 function get_varphi(obj_val,x_lr,xl_r,xu_r,x_ur,mu)
     varphi = obj_val
-    for i=1:length(x_lr)
-        xll = x_lr[i]-xl_r[i]
-        @inbounds xll < 0 && return Inf
-        @inbounds varphi -= mu*log(xll)
+    @simd for i=1:length(x_lr)
```

Comment:
Do we observe a difference when using `@simd`?

Reply:
not quite sure, I think that will need a separate benchmark